war/src/main/scss/abstracts/_theme.scss

```diff
@@ -116,8 +116,8 @@ $semantics: (
   // Deprecated - Button primary
   --button-color--primary: var(--background);
   --btn-primary-bg: #063f61;
-  --btn-primary-bg-hover: lighten(#063f61, 7.5%);
-  --btn-primary-bg-active: lighten(#063f61, 12%);
+  --btn-primary-bg-hover: #{lighten(#063f61, 7.5%)};
+  --btn-primary-bg-active: #{lighten(#063f61, 12%)};
```

Comment:
I am far from a SCSS expert, so IDK whether this is the _right_ way to do things, but it does fix the issue.